tests/acceptance/ExportSubscribersCest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+use MailPoet\Test\DataFactories\Segment;
+
+require_once __DIR__ . '/../DataFactories/Segment.php';
+require_once __DIR__ . '/../_data/MailPoetExportList.csv';
+
+class ExportSubscribersCest {
+  function __construct() {
+    $this->search_field_element = 'input.select2-search__field';
+  }
+  function exportSubscribers(\AcceptanceTester $I) {
+    $segment_name = 'Hobbyists';
+    $segment = new Segment();
+    $segment->withName($segment_name)->create();
+    $I->wantTo('Export a list of subscribers');
+    $I->login();
+    $I->amOnMailPoetPage('Subscribers');
+    //import unique user list to export, so we have exact data regardless of other tests
+    $I->click(['xpath'=>'//*[@id="subscribers_container"]/div/h1/a[2]']);
```

Comment:
I see that you are using very specific selectors, which may be brittle. I'd suggest to make them a little simpler, and here's how:

In XPath, `x/y` first selects `x`, then attempts to look for its direct child `y`.
Doing so may be inconvenient with deeper structures like one above, so you have to have a div, that has a div in it, that has a h1 in it, that has 2 `a` tags in it.

Instead, `//` looks for any node from the current one, no matter where they are. This would include not only direct children, but deeper elements as well (grandchildren, grandgrandchildren and so on).
And you can use `//` selector multiple times in the XPath query.
E.g. the query above could then be simplified to:
```
//*[@id="subscribers_container"]//a[2]
```
Which translates to: "find an element with an attribute id with value `subscribers_container`, and look for its descendants that have the `a` tag, take the second one.

So it's not always necessary to write out the whole absolute path to find what you need to click.


The second improvement you could consider is `$context` attribute of the `$I->click()` method.
Normally we use `$I->click('Some button text')` to look for specific text in the interface, right? But we can't always reliably find it, if there are multiple instances of the same text, just for different purposes (e.g. Delete button appearing for many subscribers, but we want to delete only one).
If we can provide the `$context` argument to specify in which element(s) to look for the selector.

E.g. `$I->click(['xpath'=>'//*[@id="subscribers_container"]/div/h1/a[2]']);` could be rewritten as:
```
$I->click('Export', ['xpath' => '//*[@id="subscribers_container"]//h1']);
```
Which translates to: Click on the element with text "Export" within `<div id="subscribers_container">`.

Or, since we see that the subscriber export button has the `#mailpoet_export_button` ID, we could directly select that:
```
$I->click('#mailpoet_export_button');
```